schema/library.json

```diff
@@ -1558,5 +1558,32 @@
             "        pass",
             "end"
         ]
+    },
+    "VAVTurndown": {
+        "library_item_id": 68,
+        "description_brief": "When a VAV box is in reheat mode, VAV max airflow rate must not be greater than the min design turndown ratio",
+        "description_index": [
+            "Section 6.5.2.1 ASHRAE 90.1-2016"
+        ],
+        "description_datapoints": {
+            "T_AHU_discharge": "AHU discharge air temperature",
+            "T_VAV_discharge": "VAV discharge air temperature",
+            "V_dot_VAV": "VAV airflow rate",
+			"V_dot_VAV_max": "VAV max airflow rate",
+			"VAV_min_turndown_design": "design VAV box min turndown ratio",
+			"turndown_tol": "VAV turndown tolerance"
+        },
+        "description_verification_type": "rule-based",
+        "assertions_type": "pass",
+        "description_assertions": [
+            "if T_VAV_discharge - T_AHU_discharge >= 5.56:",
```

Comment:
This would have to also to be modified accordingly.

Reply:
Addressed.